rules/cloud/azure/azure_app_privileged_permissions.yml

```diff
@@ -12,7 +12,7 @@ tags:
     - attack.t1098.003
 logsource:
     product: azure
-    service: microsoft365portal
+    service: auditlogs
```

Comment:
Update the logsource requires an update to the modified field. So please update it.

Reply:
Oh! My bad :) Fixed it for the three.

Reply:
No need for the other 2 :) 
I only highlighted this one because it has a changed logsource.

Reply:
Yep, I did it _via_ a comment-ammend-push-force to avoid useless modifications. All files are changed already.

Reply:
OK saw your commit, my bad. was indeed a new field `modified`.